packages/api-client/src/components/ImportCollection/ImportCollectionModal.vue

```diff
@@ -187,11 +191,18 @@ const themeStyleTag = computed(
 )
 
 function handleImportFinished() {
-  if (shouldShowIntegrationIcon.value && props.integration) {
+  // If the integration is not a valid theme id, set the theme to default
+  const integrationThemeId = themeIds.includes(
+    props.integration as IntegrationThemeId,
+  )
+    ? props.integration
+    : 'default'
```

Comment:
```ts
const isIntegrationThemeId = (value: unknown): value is IntegrationThemeId => 
  themeIds.includes(value as IntegrationThemeId);
```

```suggestion
  // If the integration is not a valid theme id, set the theme to default
const isIntegrationThemeId = (value: unknown): value is IntegrationThemeId => 
  themeIds.includes(value as IntegrationThemeId);

const integrationThemeId = isIntegrationThemeId(props.integration) 
  ? props.integration 
  : 'default';
```
lets move away from using `as` and instead use a type guard here

@geoffgscott should we ban the use of `as`?

Reply:
yes better approach to prevent runtime error actually